content/posts/May-2022.md

```diff
@@ -24,3 +24,9 @@ weight: 62
   ([packit#1563](https://github.com/packit/packit-service/pull/1563))
 - Packit now supports `release_suffix` configuration option that allows you to override the long release string provided by Packit that is used to ensure correct ordering and uniqueness of RPMs built in Copr.
   ([packit#1568](https://github.com/packit/packit-service/pull/1568))
+
+## Week 18 (May 3rd - May 9th)
+
+- packit source-git commands learnt to replace Git-trailers in commit messages if they already exist. ([packit#1577](https://github.com/packit/packit/pull/1577))
```

Comment:
The script seems to drop formatting :)

```suggestion
- `packit source-git` commands learnt to replace [Git-trailers in commit messages](https://packit.dev/source-git/work-with-source-git/control-patch-generation/) if they already exist. ([packit#1577](https://github.com/packit/packit/pull/1577))
```

Reply:
I have a feeling it's dropped by Zuul already and the messages are taken from commits as opposed to PRs.

I think we should also include dashboard :D

Reply:
:+1: